src/AccessibilityInsights.Automation/ExecutionWrapper.cs

```diff
@@ -25,6 +25,7 @@ internal static T ExecuteCommand<T>(Func<T> command, Func<string, T> errorFactor
                 }
                 catch (Exception ex)
                 {
+                    // No need to report this Exception, since telemetry is disabled in automation
```

Comment:
Disabled as in not possible or not desirable?

Reply:
Updated the comment to say "telemetry is not available in automation"